lib/northbound_sysrepo.c

```diff
@@ -186,12 +186,12 @@ static int frr_sr_process_change(struct nb_config *candidate,
 	/* Map operation values. */
 	switch (sr_op) {
 	case SR_OP_CREATED:
+		nb_op = NB_OP_CREATE;
+		break;
```

Comment:
This is no longer behing checked for validatity/allowed

Reply:
`NB_OP_CREATE` is always allowed, even for list keys, so there's no need for additional checks.
There's still a check in `SR_OP_MODIFIED` case to ignore changes to list keys.